api.md

```diff
@@ -58,6 +58,52 @@ your use case. Users will *rarely* want or need to use this command.
 converted to a hypertable (via `create_hypertable`), but must similarly
 be run only on an empty hypertable.
 
+**Space partitions**: The use of additional partitioning is a very
+specialized use case.  **Most users will not need to use it.**
+
+Space partitions use hashing: Every distinct item is hashed to one of
+*N* buckets.  Remember that we are already using (flexible) time
+intervals to manage chunk sizes; the main purpose of space
+partitioning is to enable parallel I/O to the same time interval.
+
+Parallel I/O can benefit in two scenarios: (a) two or more concurrent
+queries should be able to read from different disks in parallel, or
+(b) a single query should be able to use query parallelization to read
+from multiple disks in parallel.
+
+Note that query parallelization in PostgreSQL 9.6 (and 10) does not
+support querying *different* hypertable chunks in parallel;
+query parallelization only works on a single physical table (and thus
+a single chunk). We might add our own support for this, but it is not
+currently supported.
+
+Thus, users looking for parallel I/O have two options:
+
+1. Use a RAID setup across multiple physical disks, and expose a
+single logical disk to the hypertable (i.e., via a single tablespace).
+
+1. For each physical disk, add a separate tablespace to the
+database.  TimescaleDB allows you to actually add multiple tablespaces
+to a *single* hypertable (although under the covers, each underlying
+chunk will be mapped by TimescaleDB to a single tablespace / physical
```

Comment:
Realize that this is not new text, but find this potentially misleading when it says that each chunk is mapped to a single tablespace. This could mean, either (1) all chunks end up on the same tablespace, or (2) a chunk is not split across multiple tablespaces (which is not really possible).

What I think we intend to say here is simply that a hypertable's chunks are spread across the tablespaces associated with that hypertable.

Reply:
Adjusted with that verbage